smack-extensions/src/main/java/org/jivesoftware/smackx/geoloc/packet/GeoLocation.java

```diff
@@ -308,126 +335,255 @@ public static GeoLocation from(FormField formField) {
         private String tzo;
         private URI uri;
 
+        /**
+         * Sets accuracy.
+         * @param accuracy : Horizontal GPS error in meters.
```

Comment:
For all those builder method documentations:
I'd probably put the detailed description of the parameter into the header of the documentation and replace the `@param` documentation with a short name.

Eg.
```
/**
 * Sets the accuracy of the GPS error in meters.
 * @param accuracy accuracy in meters
 * @return Builder
 */
```